packages/components/src/Field-Next/TextField.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import type { Meta, ComponentStory } from '@storybook/react';
+import { TextField } from './TextField';
+
+export default {
+  title: 'Components/TextField',
+  argTypes: {
+    label: {
+      control: {
+        type: 'text',
+      },
+      description: 'The label',
+      defaultValue: 'Label',
+    },
+    description: {
+      control: {
+        type: 'text',
+      },
+      description: 'Help Text',
+      defaultValue: 'This is a help text description',
+    },
+    error: {
+      control: {
+        type: 'boolean',
+      },
+      description: 'Is the input invalid?',
+      defaultValue: false,
+    },
+    errorMessage: {
+      control: {
+        type: 'text',
+      },
+      description: 'Error Message',
+      defaultValue: 'Something went wrong',
+    },
+    required: {
+      control: {
+        type: 'boolean',
+      },
+      defaultValue: false,
+    },
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+      defaultValue: false,
+    },
+    readOnly: {
+      control: {
+        type: 'boolean',
+      },
+      defaultValue: false,
+    },
+    type: {
+      control: {
+        type: 'select',
+      },
+      options: [
+        'date',
+        'datetime-local',
+        'email',
+        'month',
+        'number',
+        'password',
+        'search',
+        'tel',
+        'text',
+        'time',
+        'time',
+        'url',
+        'week',
+      ],
+      defaultValue: 'text',
+      table: {
+        defaultValue: {
+          summary: 'text',
+        },
```

Comment:
I would add defaultValue prop also to storybook

Reply:
Hm, why? Does storybook reload when pros change?

Reply:
no just to test the prop in storybook

Reply:
then a `defaultValue` shouldn't  work since it's only applied on first render

Reply:
ok